.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,258 @@
+
+    Args:
+        filepath: Path to the translation file
+
+    Returns:
+        translation: Loaded translation
+    """
+    try:
+        with open(filepath, encoding="utf-8") as file:
+            content = file.read()
+            if not content.strip():
+                raise ValueError(f"File {filepath} is empty.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)